railgun/test/integration/tcpingress_test.go

```diff
@@ -42,6 +50,9 @@ func TestMinimalTCPIngress(t *testing.T) {
 		assert.NoError(t, cluster.Client().CoreV1().Services(namespace).Delete(ctx, service.Name, metav1.DeleteOptions{}))
 	}()
 
+	// TODO: this is a workaround for https://github.com/Kong/kubernetes-ingress-controller/issues/1146
```

Comment:
note: I added an acceptance criterion to the referenced issue, to deal with this TODO